src/app/reducers/index.js

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from 'redux';
 import FireBaseUserReducer from './firebase_user_reducer';
+import UserCourses from './databse'
```

Comment:
No reason, I typed it fast missed the "a" but only realized after I did a bulk of the work. I plan on changing it when a reducer for the course description is created.